src/classes/CRLP_RollupAccount_TEST.cls

```diff
@@ -36,9 +36,9 @@
 @IsTest
 private class CRLP_RollupAccount_TEST {
 
-    /** @description The 5 testing types supported by the test engine */
+    /** @description The 6 testing types supported by the test engine */
     private Enum TestType {
-        TestTrigger, TestQueueuable, TestBatch, TestSkewBatch, TestMultiCurrBatch
+        TestTrigger, TestQueueuable, TestBatch, TestSkewBatch, TestMultiCurrBatch, TestReparentOpp
```

Comment:
Formatting: should be lower case enum and UPPER_CASE values. I realize it is not part of this bug fix, but thought it would be worth mentioning in case you want to adjust it while you are in here. Thanks!

Reply:
I thought about this, but there are 4 test classes that all follow this same pattern. I'd rather tackle them all at once, but I can modify this one if you prefer.